post-payments.php

```diff
@@ -196,7 +201,9 @@ public function add_author( $post_author, $post_id, $data = null ) {
 	}
 
 	public function format_currency( $number ) {
-		return $this->currency_symbol . number_format( $number, 2 );
+		if ( ! empty( $number ) ) {
+			return $this->currency_symbol . number_format( $number, 2 );
```

Comment:
The number should be localized using [`number_format_i18n()](https://codex.wordpress.org/Function_Reference/number_format_i18n)`. If the currency symbol is a string of letters like USD, there should be a space after it.